components/core/src/utils/make_dictionaries_readable/make-dictionaries-readable.cpp

```diff
@@ -1,6 +1,6 @@
 // C++ standard libraries
 #include <string>
-
+#include <queue>
```

Comment:
Mistakenly added to the pr. but I personally found it helpful to also dump out the segindex in human readable format for my debugging so I just leave it included.